bigquery_etl/cli/query.py

```diff
@@ -226,20 +268,108 @@ def create(ctx, name, sql_dir, project_id, owner, dag, no_schedule):
             owners=[owner],
             labels={"owner": safe_owner},
         )
-
         view_metadata.write(view_metadata_file)
 
+        click.echo(f"Created view in {view_path}")
+
     # create query.sql file
-    query_file = derived_path / "query.sql"
-    query_file.write_text(
-        reformat(
-            f"""-- Query for {dataset}.{name}{version}
-            -- For more information on writing queries see:
-            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
-            SELECT * FROM table WHERE submission_date = @submission_date"""
+    if use_live:
+        macro_file = path / project_id / dataset / (table_name + "_macros.jinja")
+        if hourly:
+            macro_file.write_text(
+                reformat(
+                    f"""{{% macro {table_name}(use_live) %}}
+                        SELECT
+                            *
+                        FROM
+                            {{% if use_live %}}
+                            table_live
+                            {{% else %}}
+                            table_stable
+                            {{% endif %}}
+                        WHERE
+                            {{% if use_live %}}
+                            TIMESTAMP_TRUNC(submission_timestamp, HOUR) = @submission_hour
+                            {{% else %}}
+                            TIMESTAMP_TRUNC(submission_date, DAY) = @submission_date
+                            {{% endif %}}
+                        {{% endmacro %}}"""
+                )
+                + "\n"
+            )
+        else:
+            macro_file.write_text(
+                reformat(
+                    f"""{{% macro {table_name}(use_live) %}}
```

Comment:
Because we can only partition by day or by hour, there isn't really a simple standard SQL for this case. This is what I came up with, but I'm open to other suggestions